src/cdk-experimental/listbox/listbox.ts

```diff
@@ -178,12 +182,18 @@ export class CdkOption implements ListKeyManagerOption, Highlightable {
       '[attr.aria-activedescendant]': '_getAriaActiveDescendant()'
     }
 })
-export class CdkListbox implements AfterContentInit, OnDestroy, OnInit {
+export class CdkListbox implements AfterContentInit, OnDestroy, OnInit, ControlValueAccessor {
```

Comment:
```suggestion
export class CdkListbox<T> implements AfterContentInit, OnDestroy, OnInit, ControlValueAccessor {
```
And all references to `CdkOption` throughout should be `CdkOption<T>`